security/secret-protection/docs/secret-protection.rst

```diff
@@ -0,0 +1,281 @@
+
+.. _secret-protection-tutorial:
+
+Tuturial: Secret Protection
```

Comment:
wrt TOC, what do you think about adding it under https://docs.confluent.io/current/security/secrets.html (requires PR to docs)

Reply:
Yes I think that's a good place.

Reply:
Cool.  Can you create a PR to docs to add it there?

Reply:
Yes will do.